This code allows POV-Ray to complain about "beta out of date", but stops it from exit(int)'ing
--- povray-3.7.0.beta.29/vfe/unix/unixconsole.cpp	2008-08-04 12:52:56.000000000 -0400
+++ /tmp/buffer-content-10089Ugl	2008-09-26 22:03:58.000000000 -0400
@@ -272,7 +272,7 @@
 				if (argv[i] != NULL && strcmp(argv[i], "--betacode") == 0)
 				{
 					fprintf(stderr, "%d\n", current_code);
-					exit(RETURN_OK);
+					//exit(RETURN_OK);
 				}
 			const char *beta_code = getenv("POVRAY_BETA");
 			if (beta_code != NULL)
@@ -285,7 +285,7 @@
 				PACKAGE,
 				current_week < 52 ? "has expired" : "is over twelve months old and is no longer usable"
 			);
-			exit(RETURN_ERROR);
+			//exit(RETURN_ERROR);
 		}
 		current_time = EXPIRE_AT + (current_week+1)*spw - current_time;
 	}

Diff finished.  Fri Sep 26 22:03:58 2008
